docs/examples/jobs/job.yaml

```diff
@@ -0,0 +1,7 @@
+controller: job
+containers:
+- image: perl
+  name: pival
```

Comment:
why don't you use top level name here?

Reply:
I have thought that top level name should be required field.

Reply:
Have you tried to run this example?
I get the following error because of the missing name.

```
▶ ./kedge generate -f test.yaml | kubectl create -f -
The Job "" is invalid: metadata.name: Required value: name or generateName is required
```

Reply:
fixed